modules/imgproc/src/resize.cpp

```diff
@@ -95,6 +95,8 @@ static void hlineResize(ET* src, int cn, int *ofst, FT* m, FT* dst, int dst_min,
             }
         }
     }
+    // Avoid reading beyond the end of the source image.
+    if (i >= dst_width) return;
     ET* src_last = src + cn*ofst[dst_width - 1];
     for (; i < dst_width; i++) // Points that fall right from src image so became equal to rightmost src point
```

Comment:
> `// Avoid reading beyond the end of the source image.`

How could it be read? There is `for`'s condition which prevents that.

Reply:
I re-worded the comment. ofset is the one that is unset, leading to a (useless) random memory read.